common/test/acceptance/tests/helpers.py

```diff
@@ -354,6 +359,92 @@ def verify_events_of_type(self, username, event_type, expected_events, expected_
                     "Refer '{0}' does not have correct suffix, '{1}'.".format(actual_referer, expected_referers[index])
                 )
 
+    @contextmanager
+    def capture_events(self, event_filter=None, number_of_matches=1, captured_events=None):
```

Comment:
Yeah I wanted to make sure the pattern makes sense before going too nuts with docs etc.